src/popover/popover.ts

```diff
@@ -96,6 +103,33 @@ export class NgbPopover implements OnInit, OnDestroy {
     });
   }
 
+  @HostListener('document:click', ['$event'])
+  /**
+   * Determines if a click occurs outside of the popover.
+   */
+  private _docClickListener(e) {
```

Comment:
We should try to name this method in a descriptive way instead of adding a comment on top of it.